resources/functionMap.php

```diff
@@ -377,7 +377,7 @@
 'asinh' => ['float', 'number'=>'float'],
 'asort' => ['bool', '&rw_array_arg'=>'array', 'sort_flags='=>'int'],
 'assert' => ['bool', 'assertion'=>'string|bool', 'description='=>'string|Throwable|null'],
-'assert_options' => ['mixed', 'what'=>'int', 'value='=>'mixed'],
+'assert_options' => ['object|array|string|int|null', 'what'=>'int', 'value='=>'mixed'],
```

Comment:
There's definitely missing `false` here, see the docs https://www.php.net/manual/en/function.assert-options.php

Reply:
But at this point, the `mixed` should stay as basically any type can be returned...

Reply:
Changes to ```assert_options``` have been rolled back.